GameDev/week6/livestream/outline.md

```diff
@@ -0,0 +1,30 @@
+Outline:
+
+Hey all, today we’ll be looking at how to put sounds in your Unity projects. I’ll be going over how to play sounds through code, how to adjust those sounds in Unity, and how to find and best use free sound assets online.
+
+Now, hopefully you’ve already downloaded and opened the same project I have in front of me, which you can find and download at this github link: https://github.com/chris8736/bit_temp_mds/tree/main 
+
```

Comment:
Same issue here with the bullet points. Use the "preview" option to see how the markdown works, and [here's](https://www.markdownguide.org/cheat-sheet/) a cheatsheet on how to use markdown! (it's kind of like a formatting coding language)